bluesky/tests/test_scans.py

```diff
@@ -479,3 +479,22 @@ def test_relative_fermat_spiral(RE, hw):
     approx_multi_traj_checker(RE, scan,
                               _get_fermat_data(start_x, start_y),
                               decimal=2)
+
+
+def test_x2x_scan(RE,hw):
```

Comment:
Style thing: needs a space after this comma. (Soon we'll probably apply a linter, as many large project do, so these kinds of things we'll actually be _tested_.)

Reply:
Yes, there is a lot of cleaning up to do on all of our projects. I have a branch (somewhere...) that attempted to do this for bluesky but it's probably hopelessly out of date now. Might as well start over.